src/counter_analysis_toolkit/src/Makefile

```diff
@@ -66,7 +66,7 @@ all: branch.o d_cache eventstock.o flops i_cache instr vector
 
 d_cache: timing_kernels.o prepareArray.o compar.o dcache.o
 
-i_cache: icache.o icache_seq_kernel_0.o
+i_cache: icache.o icache_seq_kernel.o
```

Comment:
What was the significance of the zero originally in the name? As it is now discarded.

Reply:
Got it. Thank you for the clarification!